tests/accessor/accessor_common.h

```diff
@@ -608,14 +663,14 @@ void check_common_constructor(GetAccFunctorT get_accessor_functor,
  * @tparam Dimension Dimensions of the accessor
  * @tparam AccessMode Access mode of the accessor
  * @tparam Target Target of accessor
- * @tparam GetAccFunctorT Type of functor for accessor creation
```

Comment:
why did you delete this? `GetAccFunctorT` still using this, at the [this](https://github.com/KhronosGroup/SYCL-CTS/pull/339/files/b9e821d9187892d14edd647c5ce13bcf90585588..162f97c3bfafe3ff452eda18ab31e536c6b8e5f9#diff-8067235ffc00be4a68e8da8448acbf789b606099c5dbc1e969df066646bd2f64R671) line